genomicFileHandler/read_info_extractor.py

```diff
@@ -164,46 +164,60 @@ def pileup_indel_DP4(pileup_object, indel_pattern):
         ref_rev = pileup_object.reads.count(',')
         alt_for = pileup_object.reads.count( indel_pattern.upper() )
         alt_rev = pileup_object.reads.count( indel_pattern.lower() )
-        
+
         dp4     = ref_for, ref_rev, alt_for, alt_rev
-        
+
     else:
         dp4 = nan,nan,nan,nan
 
     return dp4
 
 
 def pileup_DP4(pileup_object, ref_base, variant_call):
-    
+
     base_calls = pileup_object.base_reads()
-    
+
     if base_calls:
-        
+
         # SNV
         if len(variant_call) == len(ref_base):
-            
+
             ref_for,ref_rev,alt_for,alt_rev = base_calls[0], base_calls[1], base_calls[2].count(variant_call.upper()), base_calls[3].count(variant_call.lower())
-        
+
         # Insertion:
         elif len(variant_call) > len(ref_base):
-            
+
             inserted_sequence = variant_call[ len(ref_base):: ]
-            
+
             ref_for,ref_rev,alt_for,alt_rev = base_calls[0], base_calls[1], base_calls[6].count(inserted_sequence.upper()), base_calls[7].count(inserted_sequence.lower())
-        
+
         # Deletion:
         elif len(variant_call) < len(ref_base):
-            
+
             deleted_sequence = ref_base[ len(variant_call):: ]
-            
+
             ref_for,ref_rev,alt_for,alt_rev = base_calls[0], base_calls[1], base_calls[4].count(deleted_sequence.upper()), base_calls[5].count(deleted_sequence.lower())
-    
+
     else:
         ref_for = ref_rev = alt_for = alt_rev = 0
-    
+
     return ref_for, ref_rev, alt_for, alt_rev
-    
-    
+
+
+
+
+def rescale(x, original=None, rescale_to=None, max_phred=1001):
+    if ( rescale_to == None ) or ( original.lower() == rescale_to.lower() ):
+        y = x if isinstance(x, int) else '%.2f' % x
+    elif original.lower() == 'fraction' and rescale_to == 'phred':
```

Comment:
Although `original.lower()` is being used, `original` can be `None` due to the default keyword value in this function. Should either check for `None` value or set the default value to a valid setting.